jobsdb/jobsdb_backup_test.go

```diff
@@ -191,6 +191,9 @@ func (*backupTestCase) insertRTData(t *testing.T, jobs []*JobT, statusList []*Jo
 		}
 		return jobsDB.copyJobStatusDS(context.Background(), tx, rtDS2, statusList, []string{}, nil)
 	})
+	jobsDB.dsListLock.WithLock(func(l lock.DSListLockToken) {
```

Comment:
why do we need this?

Reply:
nope, since we are anyways refreshing after the transaction. We don't need it anymore. Removed